src/devol.cpp

```diff
@@ -62,7 +63,7 @@ void devol(double VTR, double f_weight, double f_cross, int i_bs_flag,
         } else {                                // or user-specified initial member 
             ta_popP.col(i) = initialpop.col(i);
         } 
-        memcpy(REAL(par), ta_popP.colptr(i), Rf_nrows(par) * sizeof(double));      
+        memmove(REAL(par), ta_popP.colptr(i), Rf_nrows(par) * sizeof(double));      
```

Comment:
memmove() permits the situation where the memory location overlap between the origin and the destination, memcpy() doesn't. In this case memmove() would be safer than memcpy() to mitigate potential overlapping.